### ISSUE TITLE
Can't de-authorize spotify

### ISSUE THREAD

**Issue by Melodeiro** [Epicguru/RimVibesMod#5]
I have multiple spotify profiles on my desktop, and was logged in a wrong profile on my browser, which i didn't noticed. Now i can't change profile, because there is no such feature, and also after revoking access under spotify profile, mod is totally broken.
Where is the cache storage? I don't see anything under rimworld config directory, also tried regedit, but didn't found anything contains rimvibes. From the game i see this image now: 
![image](https://user-images.githubusercontent.com/2702567/95686490-c9ce2700-0c06-11eb-932d-4ca610b9c65f.png)

**Comment by Melodeiro** [comment]
Got it working by unsubscribing and removing mod directory from steam, then subscribing again. But i guess, mod needs a de-authorize button or something like that

**Comment by Epicguru** [comment]
Yes, I will add a log-out button when I get round to updating this mod. There are a few other bugs that need fixing, so it might be soon.

Thanks for the report.
